src/server.js

```diff
@@ -1,21 +1,25 @@
-const express = require('express');
+const express = require("express");
 const app = express();
 
-const cors = require('cors');
-const morgan = require('morgan');
+const cors = require("cors");
+const morgan = require("morgan");
 
-app.disable('x-powered-by');
+app.disable("x-powered-by");
 
 // Add middleware
 app.use(cors());
-app.use(morgan('dev'));
+app.use(morgan("dev"));
 app.use(express.json());
 app.use(express.urlencoded({ extended: true }));
 
-
 // Tell express to use your routers here
-const customerRouter = require('./routers/customer');
-app.use('/customers', customerRouter);
+const customerRouter = require("./routers/customer");
+app.use("/customers", customerRouter);
+
+const moviesRouter = require("./routers/movies");
+app.use("/movies", moviesRouter);
 
+const screensRouter = require("./routers/screens")
+app.use("/screens", screensRouter)
 
-module.exports = app
+module.exports = app;
```

Comment:
Overall, everything looks great, spot on work! - the only thing i would change is adding code annotation just for extra readability ( since i feel there is no other areas for improvement )